samples/CallWithChat/src/app/views/HomeScreen.tsx

```diff
@@ -139,7 +139,7 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                          *
                          * write regexp to flatten this
                          */
-                        const phoneNumber = '+' + newValue?.replace('[^0-9]', '');
+                        const phoneNumber = '+' + newValue?.replace(/\D/g, '');
```

Comment:
probably need to remove spacing here? replace(' ', '')

Reply:
this clears the spaces as well in my testing